Makefile

```diff
@@ -31,31 +31,43 @@ GO_BUILD=$(GO_CMD) build
 GO_CLEAN=$(GO_CMD) clean
```

Comment:
Nothing wrong with this file and thanks for doing it but just try to keep PRs to the point in the future

Reply:
okay.